src/app/backend/logs.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"io/ioutil"
+	api "k8s.io/kubernetes/pkg/api"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Log response structure
+type Logs struct {
+	// Pod id
+	PodId string `json:"podId"`
+	// Specific date (RFC3339) when logs started
+	SinceTime string `json:"sinceTime"`
+	// Logs string
+	Logs []string `json:"logs"`
+}
+
+// Return logs for particular pod or error when occurred.
+func GetPodLogs(client *client.Client, namespace string, podId string) (*Logs, error) {
+
+	pod, err := client.Pods(namespace).Get(podId)
+	if err != nil {
+		return nil, err
+	}
+
+	limitBytes := int64(1024 * 1024)
+	logOptions := &api.PodLogOptions{
+		Follow:     false,
+		Previous:   false,
+		Timestamps: true,
+		LimitBytes: &limitBytes,
+	}
+
+	if logString, err := handleLog(client, namespace, podId, logOptions); err == nil {
```

Comment:
Oh, this is tricky IF statement. What you normally do in Go is that inside this if you `return nil, err` and then outside this IF you have normal program flow. Please refactor, so that it looks like lines (39-42)